index-config/src/main/solr-config/uniprot-collections/taxonomy/conf/schema.xml

```diff
@@ -56,6 +63,9 @@
 		tables. <field name="replaces"/> -->
 
 	<copyField source="scientific" dest="name_sort" />
+	<copyField source="tax_id" dest="tax_id_str" />
+	<copyField source="parent" dest="parent_str" />
+	<copyField source="ancestor" dest="ancestor_str" />
 	<!-- =============================== field types =============================== -->
```

Comment:
Please ignore this change. It came from the original branch from where i cut a new branch. this code is already merged in main